hdt_monitor/settings/base.py

```diff
@@ -141,11 +152,10 @@
 # Logging
 # https://docs.djangoproject.com/en/1.9/topics/logging/#configuring-logging
 
-# Turn off default Django logging
-# https://docs.djangoproject.com/en/1.9/topics/logging/#disabling-logging-configuration
-LOGGING_CONFIG = None
+# Make sure log timestamps are in GMT
+logging.Formatter.converter = time.gmtime
 
-_DEFAULT_LOG_LEVEL = SECURE_SETTINGS.get('log_level', logging.DEBUG)
+_DEFAULT_LOG_LEVEL = SECURE_SETTINGS.get('log_level', logging.ERROR)
```

Comment:
I haven't looked through the project to see whether there are any logs of note; it looks like it was simply logging to a file, so I can't imagine VPAL was checking / using the logs for anything, but is it worth/have we checked that logging ERROR only still meets their needs? For instance, [one print statement in this PR](https://github.com/penzance/hdt_monitor/pull/5/files#diff-ae35a2d9ce5d1f0edf695876a4ef4c62840cb474ddce10fc4f3e7f05d9bc60b9R41) has been switched to `info` level, and [another](https://github.com/penzance/hdt_monitor/pull/5/files#diff-cb492fd83391f009c2676f5865a6e828e966d0a8eda2e56b0d21659917a0fe39R37) to `debug`.

Reply:
I don't know if they've ever even had access to the logs; if they need anything hopefully they'll get in touch.